CircleandCross2/Program.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+
+namespace CircleandCross2
+{
+	class Program
+	{
+
+		static void Main(string[] args)
+		{
+			try
+			{
+
+
+				Console.WriteLine("Tic Tac Toe");
+
+				string[,] array = new string[3, 3] { { "[ ]", "[ ]", "[ ]" }, { "[ ]", "[ ]", "[ ]" }, { "[ ]", "[ ]", "[ ]" } };
+
+				for (int i = 0; i < array.GetLength(0); i++)
+				{
+					for (int j = 0; j < array.GetLength(1); j++)
+					{
+						string arr = array[i, j];
+						Console.Write(arr);
+					}
+					Console.WriteLine();
+				}
+
+
+				for (int a = 1; a <= 5; a++)
+				{
+
+					Console.WriteLine("podaj pozycje 'o' w lini 'i'");
+					string tekst = Console.ReadLine();
+					int pozycja1i = int.Parse(tekst);
+					Console.WriteLine(" podaj pozycję 'o' w lini 'j' ");
+					string tekst2 = Console.ReadLine();
+					int pozycja1j = int.Parse(tekst2);
+
+					array[pozycja1i, pozycja1j] = "[o]";
+
+					for (int i = 0; i < array.GetLength(0); i++)
+					{
+						for (int j = 0; j < array.GetLength(1); j++)
+						{
+							string arr1 = array[i, j];
+							Console.Write(arr1);
+
+						}
+						Console.WriteLine();
+					}
```

Comment:
Use here a function for displaying the game board

Reply:
i do it